test/e2e/lua/dynamic_configuration.go

```diff
@@ -104,6 +112,30 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 			Expect(restOfLogs).ToNot(ContainSubstring("first sync of Nginx configuration"))
 		})
 
+		It("configuration module should read from temp file when request body > client_body_buffer_size", func() {
```

Comment:
this test should not care about which module is doing what, rather it should make sure that given a payload of backends that can not be buffered then the request to that backends should still succeed